From: Andreas Stieger <Andreas.Stieger@gmx.de>
Date: Wed, 26 Jul 2023 16:08:53 +0000
Subject: chrome/browser/ui/profile_view_utils.h:40:6: error: no template named 'vector' in namespace 'std'

[10288s] In file included from ../chrome/browser/ui/profile_view_utils.cc:5:
[10288s] ../chrome/browser/ui/profile_view_utils.h:40:6: error: no template named 'vector' in namespace 'std'
[10288s] std::vector<ProfileAttributesEntry*> GetAllOtherProfileEntriesForProfileSubMenu(
[10288s] ~~~~~^

Index: chromium-116.0.5845.42/chrome/browser/ui/profile_view_utils.h
===================================================================
--- chromium-116.0.5845.42.orig/chrome/browser/ui/profile_view_utils.h
+++ chromium-116.0.5845.42/chrome/browser/ui/profile_view_utils.h
@@ -7,6 +7,7 @@
 
 #include <string>
 #include <utility>
+#include <vector>
 
 class Profile;
 class ProfileAttributesEntry;
